caasp-istio-pilot-image/caasp-istio-pilot-image.kiwi

```diff
@@ -14,7 +14,7 @@
       derived_from="obsrepositories:/suse/sle15#15.1">
       <containerconfig
         name="caasp/v4/istio-pilot"
-        tag="%%PKG_VERSION%%"
+        tag="1.5.4"
```

Comment:
If you use hardcoded tags for whatever reason there is no need to define the
`<service name="replace_using_package_using">` section in _service file.